clang/lib/Parse/ParseOpenMP.cpp

```diff
@@ -2805,7 +2818,7 @@ StmtResult Parser::ParseOpenMPDeclarativeOrExecutableDirective(
         };
     TargetOMPContext OMPCtx(ASTContext, std::move(DiagUnknownTrait),
                             /* CurrentFunctionDecl */ nullptr,
-                            ArrayRef<llvm::omp::TraitProperty>());
+                            ArrayRef<llvm::omp::TraitProperty>(), -1);
```

Comment:
```suggestion
                            {}, -1);
```
Also, do not use magic numbers